dxr/plugins/descriptor/__init__.py

```diff
@@ -48,9 +48,24 @@ class FileToIndex(dxr.indexers.FileToIndex):
     """Do lots of work to yield a description needle."""
 
     comment_re = re.compile(r'(?:.*?/\*+)(?:\s*\*?\s*)(?P<description>.*?)(?:(?:\*+/.*)|(?:$))', flags=re.M)
-    docstring_re = re.compile(r'(?:\'\'\'|""")(?:\s*)(?P<description>.*?)(?:(?:(\'\'\'|"""))|(?:$))', flags=re.M)
+    docstring_re1 = re.compile(r'"""\s*(?P<description>[^"]*)', flags=re.M)
```

Comment:
The main reason is that if I put the or around the single/double quote parts then I would not necessarily be matching them (i.e. ''' docstrings would terminate on seeing a ") and if I put the or around the entire thing I would need two capture names and still have to check for both cases later. Also I don't have to escape many quotes this way.